Lab 5.ipynb

```diff
@@ -0,0 +1,595 @@
+      "text/plain": [
+       "<Figure size 1440x5760 with 14 Axes>"
```

Comment:
These plots could be good with either higgs or background semi-transparent so the data isn't fully overlapping